Drivers/seds/TfLunaHwas.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageFile xmlns="http://www.ccsds.org/schema/sois/seds" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.ccsds.org/schema/sois/seds seds.xsd">
+  <Package name="com/n7space/TfLunaHwas">
+    <DataTypeSet>
+      <EnumeratedDataType name="TfLunaHwas_ReadSM">
+        <LongDescription>Enum listing UART read data frame state machine states</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="TfLunaHwas_ReadSM_NotFound" value="0" />
+          <Enumeration label="TfLunaHwas_ReadSM_Head" value="1"></Enumeration>
+          <Enumeration label="TfLunaHwas_ReadSM_DisL" value="3" />
+          <Enumeration label="TfLunaHwas_ReadSM_DisH" value="4" />
+          <Enumeration label="TfLunaHwas_ReadSM_StrengthL" value="5" />
+          <Enumeration label="TfLunaHwas_ReadSM_StrengthH" value="6" />
+          <Enumeration label="TfLunaHwas_ReadSM_TempL" value="7" />
+          <Enumeration label="TfLunaHwas_ReadSM_TempH" value="8" />
+          <Enumeration label="TfLunaHwas_ReadSM_Checksum" value="9" />
+        </EnumerationList>
+      </EnumeratedDataType>
+      <ContainerDataType name="TfLunaHwas_Data" shortDescription="Lidar data container (Distance, signal strength and temperature)">
+        <EntryList>
+          <Entry name="mDistance" type="com/n7space/Hwas/Word" />
+          <Entry name="mStrength" type="com/n7space/Hwas/Word" />
+          <Entry name="mTemp" type="com/n7space/Hwas/Word" />
+        </EntryList>
+      </ContainerDataType>
+      <EnumeratedDataType name="TfLunaHwas_DataFrameErr">
+        <LongDescription>Enum listing possible rx errors</LongDescription>
+        <IntegerDataEncoding encoding="unsigned" byteOrder="littleEndian" sizeInBits="8" />
+        <EnumerationList>
+          <Enumeration label="TfLunaHwas_DataFrameErr_NoHead1" value="0" />
+          <Enumeration label="TfLunaHwas_DataFrameErr_NoHead2" value="1" />

```

Comment:
What is the difference brtween NoHead1 and NoHead2?

Reply:
Ok... Maybe this should be IncorrectHeaderByte1, IncorrectHeaderByte2, or IncorrectFirstHeaderByte, IncorrectSecondHeaderByte. Just an idea...

Reply:
NoHead1 means that we didn't received first character of the header. So, assuming that we have enabled the interrupt in the middile of the lidar message, for every character that we have received we will get a frame error NoHead1. 
NoHead2 means that we have received first character of the header, but the character received after that is not correct. It might indicate that at least one byte was lost. The header looks like this 0x59 0x59. After that there are three 16 bits integers and 8 bit crc.

Reply:
Undestood. Let's stay with Head instead of Header, but still, IncorrectFirstHeadByte would be IMHO much more readable.